examples/README.md

```diff
@@ -0,0 +1,26 @@
+# Sonic Disc Reader
+A sample Arduino sketch that demonstrates how to use the SonicDisc when attached via I2C.
+
+Primarily, it is shown how to:
+* Initiate SonicDisc measurements
+* Read data
+* Filter data based on variance
+
+# SonDisk_Proc3
```

Comment:
Choose a different, more descriptive, name maybe? Also use Dis**c** instead of Dis**k**. :smile:

Reply:
renamed to DesktopDisplay_SonicDisc
renamed SonicRing_1.pde to SonicDisc_1.pde